### PR TITLE
Updating the Windows install instructions and fixing up the assets file.

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged:  https://sz-flutter.firebaseapp.com/docs/get-started/install/windows

Feel free to ignore the changes to the assets file. I was just taking the opportunity to clean it up a bit.

### CODE REVIEW

File: src/docs/get-started/install/_get-sdk-win.md [hunk 1]
```diff
@@ -12,25 +19,52 @@
...
+    directory and locate the `flutter_console.bat` file.
+    Double-click the file to start it.
+</li>
+</ol>
 
 You are now ready to run Flutter commands in the Flutter Console!
 
-To update an existing version of Flutter,
-see [Upgrading Flutter](/docs/development/tools/sdk/upgrading).
+{{site.alert.warning}}
```
Comment (csells) [inline src/docs/get-started/install/_get-sdk-win.md]: I find this confusing. We haven't asked them to download a zip or unzip anything, so there's no real context for them to understand this comment. I vote to drop it.

File: src/docs/get-started/install/_get-sdk-win.md [hunk 2]
```diff
@@ -12,25 +19,52 @@
...
+    that requires elevated privileges).
+</li>
+<li markdown="1">Clone the repo using `git clone`.
+    The following example clones the stable channel:
+
+```sh
+git clone -b stable https://github.com/flutter/flutter.git
+```
+</li>
+<li markdown="1">In the Finder, navigate to the `flutter`
```
Comment (timsneath) [inline src/docs/get-started/install/_get-sdk-win.md]: Finder? What kind of bizarre machine is this? I searched all over my Windows machine, but couldn't find a "Finder" anywhere. I think you mean, "Windows Explorer"...

File: src/docs/get-started/install/_get-sdk-win.md (hunk 1 above)
Comment (timsneath) [reply src/docs/get-started/install/_get-sdk-win.md]: Yeah, agree -- that explains why _we_ don't go down that route, but the user doesn't need to worry about what we've discarded.

File: src/docs/get-started/install/_get-sdk-win.md [hunk 3]
```diff
@@ -12,25 +19,52 @@
...
+* From the Start search bar, enter 'env'
+  and select **Edit environment variables for your account**.
 * Under **User variables** check if there is an entry called **Path**:
-  * If the entry does exist, append the full path to `flutter\bin` using `;` as
-    a separator from existing values.
-  * If the entry does not exist, create a new user variable named `Path` with
+  * If the entry exists, append the full path to `flutter\bin` using
+    `;` as a separator from existing values.
+  * If the entry doesn't exist,
+    create a new user variable named `Path` with
```
Comment (timsneath) [inline src/docs/get-started/install/_get-sdk-win.md]: environment variable, not user variable

**Pull request merged**